src/main/java/com/cozyhome/onlineshop/userservice/controller/AuthController.java

```diff
@@ -90,8 +90,8 @@ public ResponseEntity<MessageResponse> activateUser(@RequestParam @ValidUUID Str
 		return ResponseEntity.ok().header(HttpHeaders.AUTHORIZATION, token).body(new MessageResponse("success"));
 	}
 
-	@Operation(summary = "User is sent a link to change his password",
-			description = "If user doesn't remember his password during logging, link to change his password is sent to his e-mail.")
+	@Operation(summary = "Send a password reset link to the user",
```

Comment:
I would add "to the user's email address": summary = "Send a password reset link to the user's email address"

Reply:
Ok